app/views/vacancies/index.html.haml

```diff
@@ -9,7 +9,7 @@
     = render 'filters'
   .govuk-grid-column-two-thirds
     %p.govuk-heading-m.mt0.mb1.inline-block= @vacancies.total_count_message
-    - if @vacancies.searched
+    - if @vacancies.user_search?
```

Comment:
The mobile partials should also be updated to reflect this change if possible to avoid confusion.

https://github.com/dxw/teacher-vacancy-service/blob/develop/app/views/vacancies/index.html%2Bphone.haml